.circleci/config.yml

```diff
@@ -53,7 +53,35 @@ jobs:
 
       - store_artifacts:
           path: htmlcov
-  tests_js:
+
+  lint-isort:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
```

Comment:
I have seen alias mechanism in circleci (seems a feature of yaml). We should use it no ? https://circleci.com/docs/2.0/writing-yaml/#merging-maps